examples/error_handling/handle_partial_failure.py

```diff
@@ -27,9 +27,67 @@
 
 
 def main(client, customer_id, campaign_id):
+    """Runs the example code, which demonstrates how to handle partial failures.
+
+    The example creates three Ad Groups, two of which intentionally fail in
+    order to generate a partial failure error. It also demonstrates how to
+    properly identify a partial error and how to log the error messages.
+
+    Args:
+        client:  An initialized GoogleAdsClient instance.
+        customer_id: A valid customer account ID.
+        campaign_id: The ID for a campaign to create Ad Groups under.
+    """
+    try:
+        ad_group_response = create_ad_groups(client, customer_id, campaign_id)
+    except GoogleAdsException as ex:
+        print('Request with ID "{}" failed with status "{}" and includes the '
+              'following errors:'.format(ex.request_id, ex.error.code().name))
+        for error in ex.failure.errors:
+            print('\tError with message "{}".'.format(error.message))
+
+            if error.location:
+                for field_path_element in error.location.field_path_elements:
+                    print('\t\tOn field: {}'.format(
+                        field_path_element.field_name))
+        sys.exit(1)
+    else:
+        # Check for existence of any partial failures in the response.
+        if check_if_partial_failure_exists(ad_group_response):
+            print('Partial failures occurred. Details will be shown below.\n')
+            # Prints the details of the partial failure errors.
+            print_results(client, ad_group_response)
+        else:
+            print('All operations completed successfully. No partial failure '
+                  'to show.')
+
+        # In the list of results, operations from the ad_group_operation list
+        # that failed will be represented as empty messages. This loop detects
+        # such empty messages and ignores them, while printing information about
+        # successful operations.
+        for message in ad_group_response.results:
```

Comment:
I believe this should be the part  of the printResults method too. Please see this method (https://github.com/googleads/google-ads-php/blob/master/examples/ErrorHandling/HandlePartialFailure.php#L201) -- It will print what failed and what succeeded. For those succeeded, it will print their resource names.

Reply:
Done.